build.gradle

```diff
@@ -1,10 +1,11 @@
 plugins {
+    id 'org.springframework.boot' version '2.7.1'
     id "io.spring.dependency-management" version "1.0.8.RELEASE"
 }
 
 subprojects {
 
-    group 'by.battle'
+    group 'by.issoft'
```

Comment:
the best practice is to keep group as part of the package.
so if your group is
group 'by.issoft'

then base package should be 
by.issoft.***

in this example 
by.issoft.battle-service.***